manifests/apps/jupyter/create-jupyter-notebook-quickstart.yaml

```diff
@@ -6,65 +6,73 @@ metadata:
     opendatahub.io/categories: 'Getting started,Notebook environments'
 spec:
   displayName: Creating a Jupyter notebook
-  appName: jupyterhub
-  durationMinutes: 5
-  icon: 'images/jupyterhub.svg'
-  description: This quick start will walk you through creating a Jupyter notebook.
+  appName: jupyter
+  durationMinutes: 10
+  icon: 'images/jupyter.svg'
+  description: This quick start walks you through creating a Jupyter notebook.
   introduction: |-
     ### This quick start shows you how to create a Jupyter notebook.
     Open Data Hub lets you run Jupyter notebooks in a scalable OpenShift hybrid cloud environment.
 
-    We have configured the JupyterHub spawner to support your amazing Data Scientist explorations and model deployments.
+    The Jupyter spawner supports your Data Scientist explorations and model deployments.
 
-    We just know you will absolutely love this amazing environment.  This quick start will get you working in a notebook in just a few minutes.
+    We just know you will absolutely love this amazing environment.  This quick start gets you working in a notebook in just a few minutes.
   tasks:
-    - title: Launch JupyterHub
+    - title: Create a Data Science project
```

Comment:
What fuelled this change? I'd like to know if I am missing anything part of conversations that happened elsewhere.

Reply:
I have said the functionality behind `Jupyter` tile is deprecated for a long time -- perhaps I should have clarified. The Tile itself is not deprecated. It serves as a quick access point to do "one off" Notebooks. It has value, it just is stuck in a technical "failure state". It requires us to circumvent k8s RBAC, write a ton of "server side" code to manage permissions and to avoid sharing of the code in modern flows like DS Projects.

We don't want to toss the Jupyter tile out -- we want to toss the flow out... migrate to projects (or a single project with more robust RBAC or to one project per person) -- this is all still TBD on functionality. We want to keep the Jupyter Tile itself and the goals behind it, but we got to figure out the best course of action for that.

Reply:
@MelissaFlinn instead of accessing the notebooks / starting them via a Data Science Project, can you go with the flow described at https://opendatahub.io/docs/getting-started-with-open-data-hub/#launching-jupyter-and-starting-a-notebook-server_get-started instead? Thank you.